### DISCUSSION TITLE
Option to add gap between pages in continuous scrolling mode

### DISCUSSION THREAD

**Discussion by HubKing** [Kareadita/Kavita#2974] [Ideas]
### Idea Description

I think I had suggested this before in some sort of special website for feature request, but I can't find what the site was, so I cannot check the result.

The feature I want is the option to set some margin between pages in vertical scrolling mode. This is what PDF Viewers do in continuous scrolling mode. For example:

![image](https://github.com/Kareadita/Kavita/assets/10900489/09859bf2-672f-481a-8565-3278a25f64cf)

### Idea Category

Web UI

### Before submitting

- [X] I've already searched for existing ideas before posting.

**Comment by scare376** [comment]
Please specify which reader you are requesting this for as Kavita has an image reader, epub reader, and PDF reader.

**Reply by HubKing** [reply]
Frankly, I don't know what you mean. Are there many different readers? What I meant is the web UI when you open a comics book and set vertical scrolling mode

![image](https://github.com/Kareadita/Kavita/assets/10900489/49437c0e-094f-4dc4-b93c-35c0f341ed2a)

If there are other readers that support vertical continuous scrolling on page-based-data, whether they are image, PDF, etc, I think they ALL have the option to set gap between pages. epub is basically HTML pages which don't have the concept of fixed "pages", so they don't need or can't have gaps, I think.

**Reply by scare376** [reply]
Like I said Kavita has 3 different readers. The PDF reader (used for PDFs and has a gap between pages in scroll but I don't think it's configurable), an epub reader (you are correct it uses HTML and I think it could potentially have a continuous scroll if the book was built that way), and an image/archive reader (uses cb*/raw image files. Based on the image you just posted is likely the reader you are referring to)

**Reply by HubKing** [reply]
I never tried PDF in Kavita, but it's probably the default PDF viewer in the web browser (I can think of no reason to reinventing a PDF viewer when all modern browsers come with it), and as far as I have tested, all PDF viewers put some gap between pages (as they should, in my opinion) im vertical continuous mode.
